UNR-3675 Add Auto Generate Launch Configuration Checkbox (#2258)
spatialos/UnrealGDK: CHANGELOG.md

**注意**：自虚幻引擎开发套件 v0.8.0 版本起，其日志提供中英文两个版本。每个日志的中文版本都置于英文版本之后。

## [`x.y.z`] - Unreleased

> - Replaced the "Generate From Current Map" button  from the "Cloud Deployment Configuration" window by "Automatically Generate Configuration" checkbox that if ticked, generates an up to date configuration from the current map every time a cloud deployment is started.

Reviewer: This should probably go into `Features`.

> - Replaced the "Generate From Current Map" button  from the "Cloud Deployment Configuration" window by "Automatically Generate Configuration" checkbox that if ticked, generates an up to date configuration from the current map every time a cloud deployment is started.

Reviewer 2: ```suggestion
- Replaced the **Generate From Current Map** button  from the **Cloud Deployment Configuration** window with the **Automatically Generate Launch Configuration** checkbox . If ticked, it generates an up to date configuration from the current when selecting the **Start Deployment** button.
```

looking at the code the button is now called "Automatically Generate Launch Configuration", right?

Author: Yes it is, will update it now 😄!
